src/components/Graph/graph.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import 'chartjs-adapter-moment'
+import CommentModal from '../Modals/CommentModal/CommentModal'
+import { mockHr } from './MockData/hrData'
```

Comment:
would starting with a capital be an option?

Reply:
Fixed :)